test/scripts/e2e_subs/asset-misc.sh

```diff
@@ -47,4 +47,44 @@ else
     exit 1
 fi
 
+# create asset with no manager, no freezer, and no clawback
+${gcmd} asset create --creator "${ACCOUNT}" --no-manager --no-freezer --no-clawback --name "${ASSET_NAME}" --unitname iamisc --total 1000000000000 --asseturl "${ASSET_URL}"
+
+IMMUTABLE_ASSET_ID=$(${gcmd} asset info --creator $ACCOUNT --unitname iamisc|grep 'Asset ID'|awk '{ print $3 }')
```

Comment:
It doesn't look like you use this, so I'd remove it.

Reply:
It was used for `goal asset info` to get the addresses. Is there a better way to get them? :/